readme.md

```diff
@@ -1,5 +1,9 @@
 # C# code generator for AutoRest V3
 
+We have updated the default generator to the new version `--v3`, which uses `@autorest/csharp` package and it will have few side effects:
+    * It will generate code based on the [.NET SDK guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html), which will be totally different than the code generated by `--v2` version.
```

Comment:
```
 `--v2` 
```
looks like a command line parameter that can be passed but it's not. Let's use V2 and V3